### ISSUE TITLE
Add support for raw files

### ISSUE THREAD

**Issue by veitveit** [ProtProtocols/IsoLabeledProtocol#71]
This seems to work:https://github.com/compomics/ThermoRawFileParser

**Comment by jgriss** [comment]
Yes, Thermo ported their RawFileReader and also supports linux now.

**Comment by iustin94** [comment]
What about the raw files CLI application from proteowizard? They have the GUI only for windows but it looks that the msconvert CLI can be built on linux and added in the the workflow.

**Comment by jgriss** [comment]
On Linux, msconvert can currently not use vendor libraries. This can lead to issues. There is now a Java CLI available for the Thermo reader which I think will be the best option.
